openbas-api/src/main/java/io/openbas/migration/V3_18__Add_column_section_into_settings.java

```diff
@@ -0,0 +1,18 @@
+package io.openbas.migration;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+@Component
+public class V3_18__Add_column_section_into_settings extends BaseJavaMigration {
+
+  @Override
+  public void migrate(Context context) throws Exception {
+    Connection connection = context.getConnection();
+    Statement select = connection.createStatement();
+    select.execute("ALTER TABLE parameters ADD column parameter_section varchar(255) DEFAULT 'CONFIGURATION';");
+  }
```

Comment:
On second thought, we don't need this.
We will look for a map, it is not costly in terms of performance.
And this complicates the code (need to know which Section to use)